backend/internal/dto/validations.go

```diff
@@ -28,6 +30,9 @@ func init() {
 	if err != nil {
 		panic("Failed to register custom validation for username: " + err.Error())
 	}
+	err = v.RegisterValidation("client_id", func(fl validator.FieldLevel) bool {
+		return validateClientIDRegex.MatchString(fl.Field().String())
+	})
 	err = v.RegisterValidation("ttl", func(fl validator.FieldLevel) bool {
```

Comment:
Missing error check here:
```suggestion

	err = v.RegisterValidation("client_id", func(fl validator.FieldLevel) bool {
		return validateClientIDRegex.MatchString(fl.Field().String())
	})
    if err != nil {
		panic("Failed to register custom validation for client_id: " + err.Error())
	}

	err = v.RegisterValidation("ttl", func(fl validator.FieldLevel) bool {
```

Reply:
Thanks!